db/migrate/20170203020915_add_weight_to_course_assessment_question_multiple_response_options.rb

```diff
@@ -0,0 +1,18 @@
+class AddWeightToCourseAssessmentQuestionMultipleResponseOptions < ActiveRecord::Migration
+  def change
+    add_column :course_assessment_question_multiple_response_options, :weight, :integer
+    populate_default_weights
+    change_column_null :course_assessment_question_multiple_response_options, :weight, false
+  end
+
+  def populate_default_weights
+    Course::Assessment::Question::MultipleResponse.includes(:options).find_each do |question|
+      options = question.options.sort_by(&:id)
+      first_id = options.first.id
```

Comment:
No, there's a test to make sure the question has at least 1 correct option before it can be created.